week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,7 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = false; // complete this statement
```

Comment:
This isn't quite right. If we change the contents of the array, then it may be incorrect.

We instead need to use the `.every()` method to check if every element within the `group` array is part of the `students` array.

Remember that we can check if an individual value (e.g. a group member) is part of an array (e.g. students) using the `.includes()` method